### ISSUE TITLE
Allow users to define valid UserRuntimeAttributes

### ISSUE THREAD

**Issue by SD10** [IBDecodable/IBLinter#37]
At [IBAnimatable](https://github.com/IBAnimatable/IBAnimatable) we have a ton of UIKit subclasses with `@IBInspectable` properties. Some of these are `String`'s and can be very error prone to work with.

Example:
The [border property](https://github.com/IBAnimatable/IBAnimatable/blob/master/Documentation/APIs.md#borderdesignable) accepts a `String` of either: `solid` or `dash`

If the user writes something like `dotted`, we should show an error

I think this is possible but the user has to tell us:
a) All valid keys
b) All classes/subclasses this property is used on

Probably in a `.yml` file?

That may seem like a lot of work but if `IBAnimatable` were to publish this `.yml` file on their docs, it could be reusable.

**Comment by kateinoigakukun** [comment]
The idea which validates runtime attributes looks good. But if all valid keys and classes definitions make  `.iblinter.yml`

**Comment by phimage** [comment]
decoding UserRuntimeAttributes https://github.com/IBDecodable/IBDecodable/pull/5

I am interested with this feature. I am thinking about generating the file using sourcery if @IBInspectable annotation is accessible.
_Each framework must provide its own file._

**Comment by viktorasl** [comment]
How about having a rule, similar to `custom_module_rule` which would take as a param list of files where each `IBInspectable` will provide valid values in a comment.
Since IBLiner already has `SourceKitten` as a dependency it (probably?) can be done.

```yml
...
user_runtime_attributes:
- path_to_file
...
```

```swift
// _blurEffectStyle: ExtraLight, Light, Dark
@IBInspectable var _blurEffectStyle: String? {
  didSet {
    ...

// _borderType: solid, dash
@IBInspectable var _borderType: String? {
  didSet {
    ...
```
